### PR TITLE
Cybran unit fixes

### PR DESCRIPTION
**Pull request by Garanas**
Original firing animation, where the trebuchets unpack causing the weapon to go underground:
image

Corrected firing animation, where the trebuchet unpacks properly causing the weapon to stay above ground:
image

Adjustments made by Madmax

### CODE REVIEW

**Comment by Garanas** [comment]
Battleship fixes:

image
image
image
image

**Comment by Garanas** [comment]
The wagner is a bit harder to see - you can look at the unit in the base game and see how the torpedo turret (small black thing on the turret head) is sliding over it as it rotates and aims at a target.

**Comment by Garanas** [comment]
image

Adjusted the sirene to not just use the zapper projectile, but to also actually have a zapper. With thanks to Madmax for the actual work

**Comment by Garanas** [comment]
image

Same situation as for the sirene, fixed the zapper. Again, all the work is by Madmax

**Comment by shalkya** [comment]
trebuchet is it required to have that collar ? I ask the balance team their pov on the visual (even if it doesn't impact balance)

about bs, it seems that because of the change, the zapper are triggered less often, making the unit worst at countering the missile. Is it a coincidence of testing, or your changes could have impacted that ?

**Comment by Garanas** [comment]
_trebuchet is it required to have that collar ? I ask the balance team their pov on the visual (even if it doesn't impact balance)_
What collar are you referring to? I'm also not entirely sure whether the translation is going correct in my head. Could you mark it?

_about bs, it seems that because of the change, the zapper are triggered less often, making the unit worst at countering the missile. Is it a coincidence of testing, or your changes could have impacted that ?_
I'm not sure about that - I'll need to check myself. It felt fine when I was toying around with it. With what units were you testing?

File: units/URS0302/URS0302_unit.bp [hunk 1]
```diff
@@ -687,15 +687,15 @@ UnitBlueprint {
...
+            TurretBoneMuzzle = 'Left_Muzzle01',
+            TurretBonePitch = 'Left_Muzzle01',
             TurretBoneYaw = 'Left_Turret01',
             TurretDualManipulators = false,
             TurretPitch = 0,
             TurretPitchRange = 90,
             TurretPitchSpeed = 180,
             TurretYaw = 0,
-            TurretYawRange = 95,
+            TurretYawRange = 0,
```
Comment (MadMaxFAF) [inline units/URS0302/URS0302_unit.bp:698]: this shouldn't have been altered

**Comment by MadMaxFAF** [comment]
also I can submit a revised version of the trebuchet without the "collar"

**Comment by shalkya** [comment]
Thanks MadMax.

"With what units were you testing?"
uef crusier vs cybran bs. Fire at the bs, and also try to fire behind the bs.

**Comment by Garanas** [comment]
Yes - the last PR should fix it. One of the turrets wouldn't fire due to an unintended BP change.

**Comment by MadMaxFAF** [comment]
i didn't do any balance testing as such, just dose it work with no errors testing, as it's only a cosmetic change afaik

**Comment by Garanas** [comment]
image

This is how it looks like now, with thanks to Madmax.

**Comment by Garanas** [comment]
image

Fixes a small warp that was previously visible. With thanks to Madmax.

**Comment by Garanas** [comment]
@shalkya You mentioned:
_edit : well it's not a cybran unit, but maybe you can fix #3011 while you are at it_

I've looked at it and it is a different type of fix. I do not know how to fix that from the top of my head. I'm leaving that issue as is.

**Comment by Garanas** [comment]
@Tagada14 Would it be possible to get your / the balance teams opinion on these changes? They do not change the balance as far as I am aware (after fixing what Keyser noticed), but they do affect the appearance. I'd like to merge this PR next Monday morning unless there are objections.

**Comment by Garanas** [comment]
Merged after discussing with Tagada on Discord and receiving his approval.

**Pull request merged**